src/validation/configure.rb

```diff
@@ -118,7 +129,7 @@ def question_type?(value)
 
         # White-lists the keys allowed in the configure.yaml file
         validate(valid_top_level_keys: :yaml) do |yaml|
-          (yaml.keys - [:domain, :group, :node, :questions]).empty?
+          (yaml.keys - [:domain, :self, :group, :node, :questions]).empty?
```

Comment:
Can the new `Constants::CONFIGURE_SECTIONS` (with `:questions` appended) be used here?

Reply:
This has been implemented in a latter PR: https://github.com/alces-software/metalware/pull/185/commits/3119b417c85b922743e01d46e9614be251798ccd